pages/partials/instructorInfoPanel.ejs

```diff
@@ -69,26 +69,38 @@
   <% } %>
   <% } %>
 
-  <% if (typeof assessment !== "undefined") { %>
+  <% let instructorUrlPrefix = urlPrefix;
+     let ip_assessment = null;
+     let ip_assessment_instance = null;
+     if (typeof assessment !== "undefined") {
+        ip_assessment = assessment;
+        ip_assessment_instance = assessment_instance;
+        instructorUrlPrefix = urlPrefix + '/instructor';
```

Comment:
This feels a bit fragile, because it's relying on `assessment` only being defined on student pages (which I agree is currently true, but might not be true in the future). To make it clearer what's going on, could we construct everything from `plainUrlPrefix`? And test for `assessment_instance !== "undefined"` rather than `assessment`? It feels to me logical that having an `assessment_instance` implies having an `assessment`, but at the moment we are doing it the other way around.

That is, I'm thinking about something like:
* If we have an `assessment_instance`, assume we also have an `assessment` and `course_instance` and explicitly construct the URL starting from `plainUrlPrefix`.

Reply:
Done.